src/explore-education-statistics-common/src/modules/table-tool/components/__tests__/TableHeadersForm.test.tsx

```diff
@@ -217,6 +217,8 @@ describe('TableHeadersForm', () => {
       columnAxis.getByRole('heading', { name: 'Locations' }),
     ).toBeInTheDocument();
 
+    userEvent.click(screen.getByRole('button', { name: 'Move Locations' }));
```

Comment:
Aha, was about to comment that since we're editing these tests there might be argument for converting them to the `setup ... / await user.click(element)` sort of style, but then realised of course we haven't merged the upgrade yet. Shame to create extra tech debt, but ho hum

Reply:
yeah :( I'll rebase this and the other recent changes into the React branch once it gets into dev.